delft/sequenceLabelling/wrapper.py

```diff
@@ -113,6 +113,12 @@ def __init__(self,
             self.embeddings = None
             word_emb_size = 0
 
+        if learning_rate is None:
+            if transformer_name is None:
+                learning_rate = 0.0001
```

Comment:
Any reasons to change the default RNN learning rate from 0.001 to 0.0001? 0.0001 is very very low for a RNN afaik.

Reply:
Sorry, it was a mistake